packages/marko-web-native-x/browser/track-outbound-links.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <div />
+</template>
+
+<script>
+import { outboundLink } from '../utils/gtm-events';
+
+export default {
+  props: {
+    container: {
+      type: String,
+      required: true,
+    },
+  },
+  created() {
+    const elements = document.querySelectorAll(`${this.container} a[href]`);
+    elements.forEach((element) => {
+      const url = element.getAttribute('href');
+      element.addEventListener('click', (e) => {
+        e.preventDefault();
```

Comment:
If the click event is prevented, won't the link be broken?

Reply:
The NativeX GTM container does some kind of nonsense here, the `outbound_url` event being pushed into the datalayer causes the navigation